terasoluna-tourreservation-domain/src/main/resources/META-INF/spring/terasoluna-tourreservation-codelist.xml

```diff
@@ -7,6 +7,10 @@
            http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task.xsd
        ">
 
+  <bean id="fallbackToLocale" abstract="true">
```

Comment:
- [ ] Good to move after `AbstractJdbcCodeList`. Because you should collect abstract parent bean in one place.

Reply:
Modified by 1f89b528ae66543dccff4b0e36a1a525e886fb6e
It was done before the previous comment.